docs/src/creatingModels.md

```diff
@@ -52,15 +52,15 @@ p_2 = \frac{ E }{ 2 (1+\nu) }
 
 ### `materials.modelParams`
 
-A cell structure with vectors with the material properties of each material used in the model. The $i$-th entry of the cell, contains a vector like this:
+A vector with the material properties of each material used in the model. The $i$-th entry of the vector, contains:
```

Comment:
Thanks Mauricio i would say
```suggestion
A vector with the properties of this material model.
```
The ith should be moved since now materials is a vector of structs... There might be necessary to change the header to materials(i).modelParams... And that for elements and boundaryconds too